lib/nolibc/scanf.c

```diff
@@ -16,7 +17,14 @@ uk_scanf(void *buffer,  size_t *cnt)
 	size_t bytes_total = 0, count;
 	char *buf = buffer;
 
-	count = *cnt;
+	/* Need at least two bytes: one for user input and one for
```

Comment:
Does this comment fit with the check below (`at least two bytes` != `<= 1`)?
In the case of `size = 0`, couldn't we return without reading a char from ukplat_cink?